src/ws/cockpit-motd.service.in

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Cockpit motd updater service
+Documentation=man:cockpit-ws(8)
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=-@datadir@/@PACKAGE@/motd/update-motd
```

Comment:
And I forgot, this should be a `Type=oneshot`

Reply:
thanks for this catch.  I actually had this, but then I read somewhere that oneshot was the default, so I removed it.  I forget where that somewhere was, but maybe I just misread it.